Application linked with libsmbclient.so needs also libndr.so

Patch will be announced to Samba.org

--- a/source3/libsmb/smbclient.pc.in	2023-01-16 20:23:14.457410704 +0100
+++ b/source3/libsmb/smbclient.pc.in	2023-01-16 20:23:52.138339782 +0100
@@ -6,6 +6,6 @@
 Name: smbclient
 Description: A SMB library interface
 Version: @PACKAGE_VERSION@
-Libs: @LIB_RPATH@ -L${libdir} -lsmbclient
+Libs: @LIB_RPATH@ -L${libdir} -lsmbclient -lndr
 Cflags: -I${includedir}
 URL: http://www.samba.org/
